content/courses/native-onchain-development/hello-world-program.md

```diff
@@ -205,72 +200,72 @@ use solana_program::{
 };
 ```
 
-- `AccountInfo` - a struct within the `account_info` module that allows us to
-  access account information
-- `entrypoint` - a macro that declares the entry point of the program
-- `ProgramResult` - a type within the `entrypoint` module that returns either
-  a `Result` or `ProgramError`
-- `Pubkey` - a struct within the `pubkey` module that allows us to access
-  addresses as a public key
-- `msg` - a macro that allows us to print messages to the program log
+- `AccountInfo` - A struct within the `account_info` module that allows us to
+  access account information.
+- `entrypoint` -A macro that declares the entry point of the program.
+- `ProgramResult` - A type within the `entrypoint` module that returns either a
+  `Result` or `ProgramError`.
+- `Pubkey` - A struct within the `pubkey` module that allows us to access
+  addresses as public keys.
+- `msg` - A macro that allows us to print messages to the program log.
```

Comment:
`msg` and `Pubkey` and `ProgramResult` are good. But `AccountInfo` and `entrypoint` are recursive. What is an entrypoint? What is account information?

> `entrypoint` - A macro that declares the function that receives incoming instructions and passes them on the right instruction handler.
 > `AccountInfo` - A struct that allows us to access account information like...

For example. You're a great writer I'm sure you can do better!

Reply:
Added more details as per suggestion to explain better.